src/models/Variable.ts

```diff
@@ -8,6 +8,9 @@ export interface IVariable {
   tags: string[],
 }
 
+export const MAX_VARIABLE_NAME_LENGTH = 255 as const
+export const MAX_VARIABLE_VALUE_LENGTH = 5000 as const
```

Comment:
I'm not 100% sold on these living here but I'm not sure of a better place and this seemed central enough.

Reply:
I like this. Much better than component level